tests/rptest/services/redpanda.py

```diff
@@ -30,23 +30,12 @@
 
 
 class RedpandaService(Service):
-    PERSISTENT_ROOT = "/mnt/redpanda"
+    PERSISTENT_ROOT = "/var/lib/redpanda"
     DATA_DIR = os.path.join(PERSISTENT_ROOT, "data")
-    CONFIG_FILE = os.path.join(PERSISTENT_ROOT, "redpanda.yaml")
+    CONFIG_FILE = "/etc/redpanda/redpanda.yaml"
```

Comment:
i think this is ok, but it was also useful to help catch assumptions about the path when it was in a non-standard location.

Reply:
oh, I see. Sorry, I hadn't realized that was something this would test as well. That can be exposed as a config as part of this PR as well, should I do that? or leave as followup

Reply:
let's leave it the way you have it.